File: Modulus 2/Python/PRAK204-2310817310005-MuhammadNurwahyudiAdhitama.py
PHI = 3.142857
Jari_jari, TINGGI = input("").split(" ")

Jari_jari = int(Jari_jari)
TINGGI = int(TINGGI)

VOLUME = round(PHI * Jari_jari * Jari_jari * TINGGI, 2)
LUAS = round((2 * PHI * Jari_jari * Jari_jari) + (2 * PHI * Jari_jari * TINGGI), 2)
KELILING = round(2 * PHI * Jari_jari, 2)

print("Volume: {:.2f}".format(VOLUME))
print("Luas: {:.2f}".format(LUAS))
print("Keliling: {:.2f}".format(KELILING))

File: Modulus 2/C/PRAK204-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

void main()
{   
    float phi = 3.142857;    
    float jari, tinggi, volume, luas, keliling;
    
    scanf("%f %f", &jari, &tinggi);

    printf("Volume : %.2f\n", volume = phi * jari * jari * tinggi );
    printf("Luas : %.2f\n", luas = (2 * phi * jari *jari) + (2 * phi * jari * tinggi));
    printf("Keliling : %.2f\n", keliling = 2 * phi * jari);
    
}